src/components/WeeklySummariesReport/WeeklySummariesReport.jsx

```diff
@@ -26,6 +27,7 @@ export class WeeklySummariesReport extends Component {
       error: this.props.error,
       loading: this.props.loading,
       summaries: this.props.summaries,
+      role: this.props.authRole,
```

Comment:
Does the `state.auth.user.role` data depend on `getWeeklySummariesReport` to finish before it can be accessed?
Looks like `summaries` is intended to be accessed only after `getWeeklySummariesReport` has finished. But is that also true for `role`?
I'm wondering whether `role` can be accessed directly from `props` without getting set to the state and have the same intended behavior.